src/scss/components/_resources.scss

```diff
@@ -66,7 +66,7 @@
   --type-flag: var(--bg);
   --type-flag-active: var(--callout);
   --shadow: transparent;
-  background-color: var(--feature-accent);
+  background-color: var(--feature-accent, var(--accent));
```

Comment:
I removed the color definition for --feature-accent in the config file, but added accent as the fallback. I probably don't need to keep --featured-accent here, but I thought it could come in handy in case we have certain features we may want to highlight, we can easily adjust that variable then.